apps/api-documenter/src/yaml/IYamlApiFile.ts

```diff
@@ -188,7 +188,7 @@ export interface IYamlItem {
    *
    * NOTE: Corresponds to `ItemViewModel.Examples` in DocFX.
    */
-  examples?: string[];
+  example?: string[];
```

Comment:
👍 This correctly reflects the upstream definition from [ItemViewModel.cs](https://github.com/dotnet/docfx/blob/9fc873ef3e55270a589c4cf9dc1d829a3e901b7b/src/Microsoft.DocAsCode.DataContracts.UniversalReference/ItemViewModel.cs#L168)